src/types/Todo.tsx

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+  user: User | null;
```

Comment:
Consider making the `user` property optional instead of nullable. This can make the code more readable and avoid unnecessary null checks. 🤓